app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,16 +1,34 @@
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
 import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
 
+import Card from "./Card";
+import { MENUBAR_DATA } from "./constants";
 import DeleteAlert from "./DeleteAlert";
 import NewNotePane from "./Pane/Create";
-import Table from "./Table";
+
+const convertDateToWeekdayTime = date => {
+  const dateTime = new Date(date);
+  const weekday = dateTime.toLocaleString("en-us", { weekday: "long" });
+  const time = dateTime.toLocaleString("en-US", {
+    hour: "numeric",
+    hour12: true,
+  });
+  return `${weekday}, ${time}`;
+};
+
+const calculateHoursAgo = date => {
+  const dateTime = new Date(date);
+  const currentDateTime = new Date();
+  const hours = Math.abs(currentDateTime.getTime() - dateTime.getTime()) / 36e5;
+  return Math.round(hours);
+};
```

Comment:
In BigBinary we make use of https://day.js.org/. You can use that over here. Also these utils functions ought to be moved into an `utils.js` within the closest scope that's `Notes`.

Reply:
Resolved all the above mentioned comments in PR #13